osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -264,7 +264,8 @@ public void TestImportWhenFileOpen()
 
         private string createTemporaryBeatmap()
         {
-            var temp = new FileInfo(osz_path).CopyTo(Path.GetTempFileName(), true).FullName;
+            var temp = Path.GetTempFileName() + Guid.NewGuid() + ".osz";
```

Comment:
What is this guid for?

Reply:
To guarantee the path is unique.

Reply:
It's already unique. That's what the function returns.

Reply:
Technically not because this adds additional ".osz".

Reply:
To an already unique filename. Can you provide reproduction showing where this fails?

Reply:
For example, GetTempFileName could return a name "1.tmp" when 1.tmp doesn't exists and 1.tmp.osz exists. The idea is taken from https://stackoverflow.com/questions/581570/how-can-i-create-a-temp-file-with-a-specific-extension-with-net .

Reply:
For me this seems like it would have more chance of failure, since GUID + path could be longer than the supported path length of the OS.

I'd rather not add it without cause, and I've not yet seen cause. Nothing in that SO post says that it behaves in this way anywhere.

Reply:
ok